src/bin/solang.rs

```diff
@@ -800,7 +856,7 @@ fn target_arg(matches: &ArgMatches) -> Target {
             "error: value length cannot be modified for target '{}'",
             target
         );
-        std::process::exit(1);
```

Comment:
I believe it is clearer to leave the `std::process` here. This way we know exactly what we are exiting from.

Reply:
At the top of the file we have `use std::process::exit;` so it should be clear what is meant by `exit`. TBH I am not following what the possible confusion could be. A different exit function? Or do you think something as important as a process exit deserves to be mentioned with its full namespace?

We use things from `std` all the time without the namespace, like `format!()` and `println!()`. I don't quite see why `exit()` is any different.

Reply:
I was just afraid that someone might create an `exit` function in our codebase. It was just to avoid name collision. If you don't see this problem, I have no problem with removing the `std::process` prefix.